clients/lsp-javascript.el

```diff
@@ -68,6 +68,18 @@
   :group 'lsp-mode
   :link '(url-link "https://github.com/theia-ide/typescript-language-server"))
 
+(defcustom lsp-clients-typescript-tls-path nil
+  "Path to the typescript-language-server binary."
+  :group 'lsp-typescript
+  :risky t
+  :type 'string)
+
+(defcustom lsp-clients-typescript-tsserver-path nil
+  "Path to the tsserver binary."
+  :group 'lsp-typescript
+  :risky t
+  :type 'string)
+
```

Comment:
can you put everything in one setting `lsp-clients-typescript-args`? I am planning to do restructoring in this code and this will make it easier for me. In particular, I  want to remove passing the tsserver as a param and let the typescript-language-server discover it.

Reply:
We already have a `lsp-clients-typescript-server-args` - is this what you were asking for? Or did you want to just keep `lsp-clients-typescript-tls-path` and get rid of the tsserver one?